### ISSUE TITLE
Feature: "Next steps" closing page for all tutorials.

### ISSUE THREAD

**Issue by mikeal** [ProtoSchool/protoschool.github.io#129]
At the end of a lesson we link back to the tutorials.

Instead, we should create a page full of potential next steps someone can take after learning what they've just learned. It could include:

* Documentation and blog posts about creating things with what they've learned.
* Entry points into relevant communities.
* In more advanced tutorials it could even link into places where people are hiring developers in this particular area.
* And of course, it should link in to places in the ProtoSchool project where they can contribute as well.

**Comment by NukeManDan** [comment]
Also consider:
- example links to projects/apps that have used the concept learned for a real use case (eg. PeerPad w/ CRDTs / others in IPFS Awesome:  https://ipfs.io/ipfs/QmUQs8yuYbQ7VbMpRoL6RFz4dMuK4RqCCmPmJc5fK7P4Yc/)

**Comment by terichadbourne** [comment]
@mkikeal Do we want this page accessible all the time from the main menu? So if a tutorial now has 3 lessons, it will instead have 4 and Lesson 4 will be "Next Steps," and it would be visible when you are viewing this type of listing? 

![image](https://user-images.githubusercontent.com/19171465/51275859-1ee0f880-19a1-11e9-8c6a-ca00cb3f6de2.png)

The other option would be a top secret page that's not surfaced anywhere but linked to from the last publicly visible lesson as a sort of reward for finishing (in which case we'd have to decide whether you only have to have finished the last lesson in the tutorial or all of them, and build some new code if the latter. 

I'm leaning toward making it accessible to navigate to in case anyone wants to jump right to recommendations for follow-up materials/projects. 

Thoughts?

**Comment by mikeal** [comment]
Since it isn't something you can "complete" I'm leaning towards saying no, the last page should be hidden from this particular view.

**Comment by terichadbourne** [comment]
My concern with making a next steps page appear only after you've completed a full tutorial, rather than being a visible part of the curriculum, is that users won't be able to navigate back to it. 

Per a discussion with @mikeal, we'll postpone addressing this issue until after making a decision on Vuepress (#141), which we'll be exploring the week of 2/11-2/15.

**Comment by terichadbourne** [comment]
We'll be addressing this one in the near future, with @fsdiogo taking the first stab at it. Some of the goals we discussed today: 

- The "Next Steps" or "Resources" page will be visible from the main landing page for the tutorial, as the last lesson in the list (you will not have to complete the lessons to access it)
- If the currently-last lesson is #3, this new page will have a generic label like "Resources" or "Next Steps" as opposed to being "Lesson 4" (since you're not learning anything new there)
- We'll supply some sort of structure via the components so that all of these pages take the same format, so that we don't end up with some authors using a bulleted list, some writing paragraphs, etc.
- We might consider using either consistent headers or consistent tags to apply to the list of resources. 

UX suggestions are welcome!

**Comment by fsdiogo** [comment]
For the tutorial's landing pages I have two proposals:

<img width="1000" alt="1" src="https://user-images.githubusercontent.com/33324750/57639591-25c2f500-75a8-11e9-9321-5911a7f6ee6d.png">

---

<img width="997" alt="2" src="https://user-images.githubusercontent.com/33324750/57639599-28254f00-75a8-11e9-978d-ddebb4cceaa0.png">

If you have suggestions about the copy, please fire away.

We should try to make the text concise and the same for different tutorials.

When the user enters that lesson, it is automatically passed just as the non-exercise lessons.

[Check it in Figma](https://www.figma.com/file/QpXk5Rlv5xPfcXORUO7abWhz/protoschool?node-id=0%3A1).
